src/app/admin/components/courthouses/courthouse-search-results/courthouse-search-results.component.ts

```diff
@@ -15,6 +15,7 @@ import { TableRowTemplateDirective } from '@directives/table-row-template.direct
 })
 export class CourthouseSearchResultsComponent {
   @Input() results: Courthouse[] = [];
+  @Input() isFormEmpty!: boolean;
```

Comment:
I would call this 'show' or something similar as the child knows nothing about a form